acareeg/eegip.py

```diff
@@ -161,6 +159,12 @@ def process_events_london_resting_state(raw, age, tmax=1):
     freq = raw.info["sfreq"]
```

Comment:
what is raw.info['sfreq']? i see that freqs is referenced below in the code.

Reply:
raw is a MNE raw object. These objects have a data structure info which can be accessed like a dictionary. One of the items of this dictionary is sfreq which stand for sampling frequency. For EEG-IP, that you be 500 Hz